### DISCUSSION TITLE
how to stage all or selection of untracked files

### DISCUSSION THREAD

**Discussion by qaptoR** [NeogitOrg/neogit#1574] [Q&A]
In the past whenever I tried to stage all untracked files I would get a popup message of an error. something to do with the paths being wrong.

After an update, now it just wont even respond to my attempts, and if I select multiple files with `V` it doesn't respond to an attempt to stage them.

wondering if it's because I'm on windows 10, in powershell and or using Neovide.

**Comment by zhu-jiyuan** [comment]
You just need to press s on the untracked files.

**Reply by zhu-jiyuan** [reply]
If you are like me and have never used magit, press ? in neogit, it is very useful and solves most of the problems.

**Comment by Koriit** [comment]
Happens also if git just errors. It my case it was a leftover index lock. Worth checking manually:

```sh
$ git add FILE
fatal: Unable to create '/home/user/path/to/repo/.git/index.lock': File exists.
```
